tests/libvmi/vmi.go

```diff
@@ -174,3 +174,54 @@ func baseVmi(name string) *v1.VirtualMachineInstance {
 	}
 	return vmi
 }
+
```

Comment:
in the commit message, would you explain why these functions merit this high-level location? If you publicize a function too early, other would start using it and then it would become harder to modify.